### DISCUSSION TITLE
Duda con mockito

### DISCUSSION THREAD

**Discussion by mercedesmagnelli** [dds-jv/foro#273] [Consultas generales] [answered]
Buenas noches, estaba resolviendo el TPI y me choqué con el siguiente error: 
![image](https://user-images.githubusercontent.com/62451237/121270227-afcd0d00-c897-11eb-8c80-325382db90a5.png)

En el test correspondiente, entiendo que lo que me genera el error es la callback, que es el argumento diferente (no sé si estoy mockeando de una forma incorrecta). Qué podría hacer para solucionarlo?

Aguardo la respuesta, 
Saludos

**Comment by gastonprieto** [comment]
Buenas @mercedesmagnelli 

pudiste solucionarlo? sino decinos como lo estas mockeando para ver si te podemos dar una mano, pero si, viene por ahí el problema, no coinciden las llamadas.

**Comment by gastonprieto** [comment, marked as answer]
lo cierro por tiempo, si pudiste solucionarlo o seguis con algo que ver avisanos @mercedesmagnelli
